### ISSUE TITLE
If already numpydoc format, will remove the Raises section

### ISSUE THREAD

**Issue by dadadel** [dadadel/pyment#49]
If a function is already described by a Numpydoc docstring, Pyment will remove the _raises_ section when producing Numpydoc output docstring instead of keeping as is.

The following code (from `tests/docs_already_numpydoc.py`):
```python
def func1(param1):
    """Function 1
    with 1 param

    Parameters
    ----------
    param1 : type
        1st parameter

    Returns
    -------
    string
        a message

    Raises
    ------
    KeyError
        when a key error
    """
    return "huh"
```

Will produce the patch:
```diff
--- a/docs_already_numpydoc.py
+++ b/docs_already_numpydoc.py
@@ -12,10 +12,7 @@
     string
         a message

-    Raises
-    ------
-    KeyError
-        when a key error
+    
     """
     return "huh"
``` 
This example is in the test suite `test_pyment_cases`.

**Comment by wagnerpeer** [comment]
Docstrings containing a section following the "Raises" section will result in _pyment_ correctly parsing and generating an empty diff!
This can be tested inserting such a section into the example file `tests/docs_already_numpydoc.py` like the following:

```python
def func1(param1):
    """Function 1
    with 1 param

    Parameters
    ----------
    param1 : type
        1st parameter

    Returns
    -------
    string
        a message

    Raises
    ------
    KeyError
        when a key error
    
    Notes
    -----
    Some notes
    """
    return "huh"
```

To debug the code I reactivated the already existing but commented test `tests/pyment_test_cases.FilesConversionTests.testCaseNoGenDocsAlreadyNumpydoc`. 

To fix the issue one had to calculate the correct, positive relative end index in NumpydocTools.get_list_key instead of using -1 as end index.

But because there are more issues with already existing numpydoc which gets "converted" to numpydoc I left the tests untouched and I also did not modify the `tests/docs_already_numpydoc.py` file in the way I describe above as this would also obfuscate the current issue with using "Raises" as last section.
